### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by Yilikal250**
@Yilikal250 
@DavyKn

We have checked and corrected the accessibility based on the checking criteria provided in the accessibility exercise. 
List all points that you have checked, i.e:

 - [ ] Page titles 
We have checked that there is a title that adequately and briefly describes the content of the page.
   
- [ ]  Image text alternatives** (alt text)
Every image has an alternative text.
    
- [ ] Text headings
All text that looks like a heading is marked up as a heading.
All text that is marked up as a heading is really a conceptual section heading.
    
- [ ] Color contrast
The tool shows the page in grayscale. 
The tool lets you select a text color and a background color,
   

- [ ]  Resize

   Our page is responsive and the layout works in different screen sizes.
  Text doesn't disappear or get cut off.

- [ ]  Interaction
  Our webpage is keyboard accessible 
    
- [ ]  The basic structure of the page 
headings are right above the information they apply to.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -118,7 +117,7 @@ <h1 class="desktop">Data Dashboard Healthcare</h1>
 
     <div id="third-container" class="middle-1-container1">
       <h1 class="mobile1">Profesional Art Printing Data</h1>
```
Comment (AmaduKamara) [inline index.html]: - [ ] Kindly change all `h1` tags except the first one in the headline section to `h2/h3...`. It's a good practice for a webpage to have only one `h1` tag and all subsequent headings can be h2, h3 in a hierarchical order.  For this project and for hierarchy purposes, you should have only one h1 tag on your entire page. please change all the headings for your sections to the `h2` tag 👍

File: index.html [hunk 2]
```diff
@@ -118,7 +117,7 @@ <h1 class="desktop">Data Dashboard Healthcare</h1>
...
-      <h1 class="desktop1" >Website Protfolio </h1>
+      <h1 class="desktop1" >Website Portfolio </h1>
```
Comment (AmaduKamara) [inline index.html]: - [ ] Kindly change all `h1` tags except the first one in the headline section to `h2/h3...`. It's a good practice for a webpage to have only one `h1` tag and all subsequent headings can be h2, h3 in a hierarchical order.  For this project and for hierarchy purposes, you should have only one h1 tag on your entire page. please change all the headings for your sections to the `h2` tag 👍

File: index.html [hunk 3]
```diff
@@ -58,13 +57,13 @@ <h1>My Recent Works</h1>
...
+    <img class="middle-image" src="assets/images/ImgPlaceholderu.png" alt="Image place holders " />
   </div>
 
 
   <section class="middle-section">
     <div class="middle-image1-container">
-      <img class="middle-image1" src="assets/images/bigimageplaceholder.png" alt="">
+      <img class="middle-image1" src="assets/images/bigimageplaceholder.png" alt="Image place holders">
       <div class="middle-1-container">
         <h1>Multi-Post Stories</h1>
```
Comment (AmaduKamara) [inline index.html]: - [ ] Kindly change all `h1` tags except the first one in the headline section to `h2/h3...`. It's a good practice for a webpage to have only one `h1` tag and all subsequent headings can be h2, h3 in a hierarchical order.  For this project and for hierarchy purposes, you should have only one h1 tag on your entire page. please change all the headings for your sections to the `h2` tag 👍

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️♻️♻️

Hi @Yilikal250, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereGIF

## Highlights
✅ No Linter error
✅ README is fine
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly change all the `h1` tags to `h2/h3` as specified in the review comments
- [ ] Kindly give feedback or report on all the accessibility tests you worked on and give a detailed report as requested in your project requirement
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @Yilikal250 and @DavyKn , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**